networkx/algorithms/centrality/tests/test_betweenness_centrality_subset.py

```diff
@@ -116,6 +116,45 @@ def test_diamond_multi_path(self):
         for n in sorted(G):
             assert b[n] == pytest.approx(expected_b[n], abs=1e-7)
 
+    def test_normalized_p2(self):
```

Comment:
It might be worth adding a docstring here to indicate what the test is designed to cover; i.e. the special case when `n<=2` where the betweenness centrality should always be 0.

Reply:
Thank you so much for the suggestion:) I have updated the docstring.